src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './ProductList.scss';
 
 function ProductList() {
-  return <p>Hi</p>;
+  const [productInfoList, setProductInfoList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productList.json', { method: 'GET' })
+      .then(response => response.json())
+      .then(result => setProductInfoList(result));
+  }, []);
+  return (
+    <div className="productListContainerWrap">
```

Comment:
```suggestion
    <div className="productList">
```
가장 부모태그는 컴포넌트의 이름과 동일하게 해주세요!